packages/forms-web-app/src/dynamic-forms/middleware/get-journey-response-for-appellant-final-comments.js

```diff
@@ -0,0 +1,52 @@
+const { JourneyResponse } = require('../journey-response');
+const { APPELLANT_JOURNEY_TYPES_FORMATTED } = require('../journey-factory');
+const logger = require('#lib/logger');
+// const { getUserFromSession } = require('../../services/user.service');
+const { mapDBResponseToJourneyResponseFormat } = require('./utils');
+const { ApiClientError } = require('@pins/common/src/client/api-client-error.js');
+// const { APPEAL_USER_ROLES } = require('@pins/common/src/constants');
+
+module.exports = () => async (req, res, next) => {
+	const referenceId = req.params.referenceId;
+	const encodedReferenceId = encodeURIComponent(referenceId);
```

Comment:
not sure we need this encoding anymore by the way, its a hangover from when we had / in the caseref

Reply:
encoding removed